src/constants/data.ts

```diff
@@ -28,32 +28,32 @@ export interface ExerciseType {
 
 export const EXERCISES: ExerciseType[] = [
   {
-    key: ExerciseKeys.vocabularyOverview,
+    key: ExerciseKeys.vocabularyList,
     title: labels.exercises.vocabularyList.title,
     description: labels.exercises.vocabularyList.description,
     Level: easy,
-    nextScreen: 'VocabularyOverview'
+    nextScreen: 'VocabularyList'
```

Comment:
Do we still need this? It is basically the same as the exercise anyway...

Reply:
not sure, maybe we can refactor this too, but will merge this now, and we can open a ticket if we want to.